mover-rsync-tls/server.sh

```diff
@@ -8,12 +8,19 @@ RSYNCD_CONF=/tmp/rsyncd.conf
 STUNNEL_CONF=/tmp/stunnel.conf
 STUNNEL_PID_FILE=/tmp/stunnel.pid
 PSK_FILE=/keys/psk.txt
-STUNNEL_LISTEN_PORT=8000
 RSYNC_LOG=/tmp/rsyncd.log
+IPV6_ENABLED=$(cat /sys/module/ipv6/parameters/disable)
 
 SCRIPT_DIR="$(dirname "$(realpath "$0")")"
 cd "$SCRIPT_DIR"
 
+# If IPv6 is in enabled state, the output would be "0"
+if [ $IPV6_ENABLED -eq 0 ]; then
```

Comment:
Could you use double brackets here? this could get caught by the linter.

i.e. ```if [[ $IPV6_ENABLED -eq 0 ]]; then```